src/main/java/me/paypur/tconjei/jei/ArmorStatsCategory.java

```diff
@@ -97,8 +97,16 @@ public void draw(ToolStatsWrapper recipe, IRecipeSlotsView recipeSlotsView, Pose
             PlatingMaterialStats plating = platingStats.get();
             drawShadow(stack, String.format("[%s]", getPattern("stat.tconstruct.plating")), 0, lineNumber++, MATERIAL_COLOR);
 
-            String durabilityText = plating.getLocalizedInfo().get(0).getString().split(":")[0] + ": ";
-            String armorText = plating.getLocalizedInfo().get(1).getString().split(":")[0] + ": ";
+            String durabilityText, armorText;
+            if (plating.getLocalizedInfo().get(0).getString().contains(":")) {
+                durabilityText = plating.getLocalizedInfo().get(0).getString().split(":")[0] + ": ";
+                armorText = plating.getLocalizedInfo().get(1).getString().split(":")[0] + ": ";
+            } else if (plating.getLocalizedInfo().get(0).getString().contains("：")) {
+                durabilityText = plating.getLocalizedInfo().get(0).getString().split("：")[0] + "： ";
+                armorText = plating.getLocalizedInfo().get(1).getString().split("：")[0] + "： ";
+            } else {
```

Comment:
I think we should extract the string splitting code into its own function because its also used in `ArmorStat`, and that code hasn't been fixed

Reply:
Also in `getStatTooltip`

Reply:
I have done them